modular_skyrat/modules/mapping/code/space.dm

```diff
@@ -172,7 +172,7 @@
 	name = "Port Tarkon"
 	id = "escapefromtarkon"
 	description = "An ambitious goal, A step forward, A trial run for the Tarkon drill, ment to implant mining stations within meteors. Decades of disaster have, however, left this one... Unattended for far too long."
-	always_place = TRUE
+	always_place = FALSE
```

Comment:
What is this about?

Reply:
SDMM screenshot of port tarkon https://i.imgur.com/BSil3qo.png

Reply:
Makes it so it isn't guaranteed to spawn.

Reply:
oh god that's a whole config file. I'm not touching that. made a commit to fix it.